kotlin-jvm/versions.properties

```diff
@@ -74,3 +74,14 @@ version.retrofit2=2.9.0
 version.sqldelight=1.4.3
 
 version.com.github.ajalt..mordant=1.2.1
+
+version.org.koin..koin-core=2.1.6
```

Comment:
Can you please run `$ ./gradlew refreshVersions`?
That would re-order the keys in `versions.properties` automatically.
Then remove the comment about libraries updates if it doesn't make sense to upgrade

Reply:
Ahhh, nice. I'll run it. The comment with the version appeared here automagically o_o

Reply:
@douglascamata 
Yes, that's the main feature of https://github.com/jmfayard/refreshVersions
It lookup available dependencies updates for you, put it as comment in a way that it makes it easy to upgrade.
But the comment are not meant to be kept in git

You will have to merge `origin/main`, there is a (hopefully simple to resolve) merge conflict

Reply:
Resolved, I took care of the merge conflicts